image-swipe.android.ts

```diff
@@ -233,7 +244,7 @@ class ZoomImageView extends android.widget.ImageView {
     private _orientationChangeListener: OrientationListener;
     private _onCanScrollChangeListener: OnCanScrollChangeListenerImplementation;
 
-    constructor(context: android.content.Context) {
+    constructor(private _host: ImageSwipe, context: android.content.Context) {
```

Comment:
I think it is best for the `_host` to be changed to `WeakRef<ImageSwipe>` to avoid reference and GC problems. Also in this case I think we can eliminate the `context` param and pull it directly from `_host`?